plugins/tekton/src/components/common/Status.tsx

```diff
@@ -2,7 +2,10 @@ import React from 'react';
 
 import {
   BanIcon,
+  CheckCircleIcon,
```

Comment:
why we have two `Status` and `StatusIcon`, both look similar?

Reply:
The `Status` component was for displaying status icon and text and the `StatusIcon` component was to display only the icon. However, I have created a common component. PTAL